tests/test_fieldtypes.py

```diff
@@ -589,6 +589,41 @@ def test__is_windowslike_path(path_, is_windows):
     assert _is_windowslike_path(path_) == is_windows
 
 
+@pytest.mark.parametrize(
+    "path_, expected_str",
+    [
+        (None, None),
+        (".", "."),
+        ("", ""),
+        (" ", ""),
+        ("  ", ""),
+        ("\t", ""),
+        ("\t\t", ""),
+        ("\r", ""),
+        ("\r\r", ""),
+        ("\n", ""),
+        ("\n\n", ""),
```

Comment:
The test is incorrect as these can all be valid paths and should not be stripped:

```python
In [1]: from pathlib import Path

In [2]: open("\t\t", "w").write("tab tab")
Out[2]: 7

In [3]: str(Path("\t\t"))
Out[3]: '\t\t'

In [4]: Path("\t\t").read_text()
Out[4]: 'tab tab'
```

Reply:
Ok, sounds logical. I was a bit unsure about this, so I awaited your feedback regarding the matter.